jdaviz/configs/specviz2d/helper.py

```diff
@@ -13,14 +13,17 @@ class Specviz2d(ConfigHelper, LineListMixin):
     """Specviz2D Helper class"""
 
     _default_configuration = "specviz2d"
+    _default_spectrum_viewer_reference_name = "spectrum-viewer"
+    _default_spectrum_2d_viewer_reference_name = "spectrum-2d-viewer"
+    _default_table_viewer_reference_name = "table-viewer"
```

Comment:
specviz2d doesn't have a table viewer, by default

Reply:
That's true. But when `specviz2d.load_data` is supplied with a value for `spectrum2d`, the default parser is `mosviz-spec2d-parser` invoked, which by default has `add_to_table==True`, so a table viewer is expected implicitly.

Reply:
Ok, then that's probably fine for now, but might be a sign that we should consider refactoring some of the parsers down the road to avoid that.